pkgs/applications/graphics/inkscape/extensions/inkstitch/default.nix

```diff
@@ -51,6 +51,11 @@ python3.pkgs.buildPythonApplication {
 
   inherit dependencies;
 
+  env = {
+    # to overwrite version string
+    GITHUB_REF = "${version}";
```

Comment:
```suggestion
    GITHUB_REF = version;
```

Reply:
adjusted in e74bd37a8c15f1c96e8f9c8e5728438dafcce4c2